Procfile

```diff
@@ -1,2 +1,2 @@
-controller: COMMAND=./bin/ephemeral-access && sh -c "EPHEMERAL_LOG_LEVEL=debug EPHEMERAL_CONTROLLER_HEALTH_PROBE_ADDR=:8989 EPHEMERAL_CONTROLLER_PORT=9091 EPHEMERAL_METRICS_ADDR=:8085 $COMMAND controller"
-backend: COMMAND=./bin/ephemeral-access && sh -c "EPHEMERAL_BACKEND_NAMESPACE=ephemeral KUBECONFIG=${KUBECONFIG:-~/.kube/config} $COMMAND backend"
+controller: COMMAND=./bin/ephemeral-access && sh -c "EPHEMERAL_LOG_LEVEL=debug EPHEMERAL_CONTROLLER_HEALTH_PROBE_ADDR=:8989 EPHEMERAL_CONTROLLER_PORT=9091 EPHEMERAL_METRICS_ADDR=:8090 $COMMAND controller"
+backend: COMMAND=./bin/ephemeral-access && sh -c "EPHEMERAL_BACKEND_NAMESPACE=ephemeral KUBECONFIG=${KUBECONFIG:-~/.kube/config} EPHEMERAL_METRICS_ADDR=:8091 $COMMAND backend"
```

Comment:
It seems that you are using the wrong variable here. As defined in the backend.go the metrics variable should be `EPHEMERAL_BACKEND_METRICS_PORT`

Reply:
Also, this config isn't required in the Procfile as you are setting the default value in the go struct already.